log4j-core/src/main/java/org/apache/logging/log4j/core/impl/ThrowableFormatOptions.java

```diff
@@ -75,7 +75,9 @@
     /**
      * The list of packages to filter.
      */
-    private final List<String> ignorePackages;
+    private final List<String> filterPackages;
+
+    private final List<String> filterStartFrames;
```

Comment:
`filterPackages` suppresses the stack frames, whereas `filterStartFrames` *truncates* the stringified stack trace after any occurrence of the given package name. Correct me if I am wrong.

IMHO, *filter* here is already used in the wrong context. It is generally used for *selecting* something, not *excluding* it, e.g., `Stream#filter()`. I would have called it `discardPackages`. That said, I am probably more than a decade late to share this great wisdom. But maybe we can do something better for `filterStartFrames`. Here I see two options:

1. Stick to the historical context and go with `filterStartFrames`.
2. Try to improve the current state with the cost of some inconsistency in the syntax. `discardAfterPackages`?